packages/kit/src/version.spec.js

```diff
@@ -0,0 +1,19 @@
+import { fileURLToPath } from 'node:url';
+import { readFileSync } from 'node:fs';
+import { assert, describe, it } from 'vitest';
+import { VERSION } from './version.js';
+
+// runs the version generation as a side-effect of importing
+import '../scripts/generate-version.js';
```

Comment:
not sure where to put this..
There is no build step involved in this repo.
I guess this is needed when running the tests and for publishing `pnpm changeset:version` should be called. Are there also other places where it is needed to update the version info?